code/GridFieldLimitItems.php

```diff
@@ -143,7 +194,7 @@ public function getManipulatedData(GridField $gridField, SS_List $dataList) {
         if(($dataList instanceof UnsavedRelationList)) return $dataList;
 
         // Not compatible with paginator.
-        if ($gridField->getConfig()->getComponentByType('GridFieldPaginator')) {
+        if ($gridField->getConfig()->getComponentByType(GridFieldPaginator::class)) {
             $this->debug('GridFieldLimitItems is not compatible with GridFieldPaginator.');
```

Comment:
As discussed, we'll need to handle these debug messages - using LoggingInterface instead of SS_Log

Reply:
Done!